server/lib/validation/form_validators/ManageApplication.js

```diff
@@ -78,6 +78,8 @@ class ManageApplication extends Validator {
     logger.info('Validating key payload data');
     this.errors = {};
     this._formatIncomingPayload(payload);
+    let restrictedIps = payload.restrictedIps.split(',');
+    let javaScriptDomains =  payload.javaScriptDomains.split(',');
```

Comment:
Can we skip these tests if they are null? I would expect this to fail if no IP or Domains are provided.
https://stackoverflow.com/questions/9843394/javascript-cannot-call-method-split-of-null

Reply:
I have moved them into a conditional block